client/my-sites/plans-features-main/plan-type-selector.tsx

```diff
@@ -40,6 +40,7 @@ export type PlanTypeSelectorProps = {
 	eligibleForWpcomMonthlyPlans?: boolean;
 	isPlansInsideStepper: boolean;
 	hideDiscountLabel: boolean;
+	redirectTo?: string | null;
```

Comment:
The prop was already implemented but not defined in the prop type. Since I didn't knew precisely what value types are accepted, I made it as permissive as possible.